components/phpunit_bridge.rst

```diff
@@ -232,6 +232,51 @@ test::
 
 And that's all!
 
+.. caution::
+
+    The ``@group time-sensitive`` annotation is equivalent to ``ClockMock::register(MyTest::class)``,
+    so if you want to get a time based function mocked into one of the source code you will need to
+    add it explicitly using ``ClockMock::register(MyClass::class)``. The ``ClockMock::register`` method
+    only create a mock of the time based functions into the same namespace as your class. So when using
+    ``time()`` you will use the mock instead of the default one.
+
+
+.. code-block:: php
+
+    namespace App;
+
+    class MyClass
+    {
+        public function getTimeInHours()
+        {
+            return time() / 3600;
+        }
+    }
+
+.. code-block:: php
```

Comment:
All the code added here should keep an extra indent to be part of the caution block.

Reply:
So I can indent also the code-block ?

Reply:
Yes!